plugins/ag-charts-generate-code-reference-files/src/executors/generate/executor.ts

```diff
@@ -36,6 +37,14 @@ async function generateFile(options: ExecutorOptions) {
         case 'docs-interfaces':
             return await writeJSONFile(options.output, typeMapper.resolvedEntries());
 
+        // Patched docs-interface for front end to consume
```

Comment:
I believe the front-end is the only consumer, so let's not cater for redundant cases?